[CONTENT] New War Events

LaurBee opened the pull request:
<!-- Write BELOW The Headers and ABOVE The comments else it may not be viewable. -->
<!-- You can view CONTRIBUTING.md for a detailed description of the pull request process. -->
<!-- Be sure to name your PR something descriptive and succinct; include Bugfix: Feature: Enhancement: or Content: in the title to describe what type of PR it is. -->
<!-- IF YOU ARE DOING A BUGFIX: Please target the latest release branch if the bug that you are fixing is also present in the latest release. -->

## About The Pull Request
Adds four new_cat war event for welcoming Clans where a loner/loners shares some of their supplies with the Clan
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage senior developers from merging your PR! -->

## Why This Is Good For ClanGen
Will add new neutral/positive war events to the game, thus lowering event repetition
<!-- If this is a bug fix, you can remove this section. -->
<!-- Please add a short description of why you think these changes would benefit the game. -->
<!-- If this PR was the result of discussion/testing on the discord, please add a link to the discord conversation here. -->
<!-- If you have multiple features that can stand on their own, or unrelated bugfixes, please create separate PRs for them. -->

## Proof of Testing
![lonergift_herbs1](https://github.com/user-attachments/assets/f0a1d69a-34af-44d0-95c3-c2b4a3c6ce3a)


<!-- Include any screenshots, debugging steps, or links to beta testing threads here. At least one form of proof of testing is REQUIRED for all new content. You must be able to run the code locally before you PR it here. -->

## Changelog/Credits
New War Events
<!-- Include any changes that should be made to the changelog of the game here or any changes to the credits file of the game. -->
<!-- This is just for easy access later for senior developers gathering this information; this process is not automated. -->

LaurBee committed:
[CONTENT] New War Event

Adds a new new_cat war event where a loner shares some of their herbs with the Clan

scribblecrumb commented on resources/lang/en/events/new_cat/general.json:
```diff
@@ -814,5 +814,43 @@
+            {
+                "type": "any_herb",
+                "trigger": [
+                    "always"
```
```suggestion
                "trigger": [
                    "low",
                    "adequate"
```

LaurBee replied:
Okie dokie 👍 Done!

scribblecrumb reviewed (changes requested):
Let's change the trigger to lower supply amounts.  That way the clan will be getting an herb they likely need.

Otherwise, this looks great!

LaurBee committed:
Edited Supply Trigger

Co-authored-by: scribblecrumb <115410010+scribblecrumb@users.noreply.github.com>

LaurBee committed:
More War Events

Added three new war events that are variants of gen_new_cat_war_lonergift_herbs1: one herb-giving event where it is three cats giving herbs instead of one, and then two variants that provide fresh-kill

Also updated my info in the game credits

LaurBee commented:
Added a few variants to the initial event!

scribblecrumb reviewed (approved):
love it!! such a fun addition
